src/App.js

```diff
@@ -41,6 +42,15 @@ function App() {
             </Page>
           )}
         />
+        <Route
```

Comment:
Details of the /blog component aren’t provided, but the structure seems correct.